Sources/StreamChat/ChatClient.swift

```diff
@@ -285,6 +127,94 @@ public class ChatClient {
         self.config = config
         self.environment = environment
 
+        let configuration = config.urlSessionConfiguration
```

Comment:
maybe a separate setup method for readability?

Reply:
This one was not possible to extract to a separate function because it needs to be configured before passing it to the `APIClient`. At this time `self` is not available yet.